eidpdata/src/main/java/uk/ac/ucl/eidp/data/PoolStrategy.java

```diff
@@ -146,7 +133,11 @@ private void initialiseConnection() {
     return fields;
   }
 
-  private void setValues(PreparedStatement ps, List<String> fields, Map<String, String> parameters, Map<String, Parameter> p) throws SQLException {
+  private void setValues(
+    PreparedStatement ps, 
```

Comment:
\[Checkstyle\] WARNING: 'PreparedStatement' have incorrect indentation level 4, expected level should be 6\.